projects/02-dataverse-explorers/README.md

```diff
@@ -0,0 +1,446 @@
+el 90% de la data que existe hoy ha sido creada durante los últimos dos años.
+Cada día generamos 2.5 millones de terabytes de datos, una cifra sin
+precedentes.
+
+No obstante, los datos por sí mismos son de poca utilidad. Para que esas
+grandes cantidades de datos se conviertan en **información** fácil de leer para
+los usuarios, necesitamos entender y procesar estos datos. Una manera simple de
+hacerlo es creando _interfaces_ y _visualizaciones_.
+
+En la siguiente imagen, podrás ver cómo con la data que que se ve en la parte
+izquierda se puede construir una interfaz amigable y entendible por las
+usuarias, al lado derecho.
+
+![pokemon-data-to-ui](https://user-images.githubusercontent.com/12631491/218505816-c6d11758-9de4-428f-affb-2a56ea4d68c4.png)
+
+## 2. Resumen del proyecto
+
+En este proyecto utilizarás herramientas de inteligencia artificial como
+[ChatGPT](https://openai.com/chatgpt), [ExplainDev](https://explain.dev/),
+entre otras para generar set de datos en formato JSON. También te familiarás
+con técnicas de generación de imágenes como `Stable Diffusion` para generar
+imágenes.
+
+Además **construirás una _página web_ para visualizar un
+_conjunto (set) de datos_** que se adecúe a lo que descubras que tu usuario
+necesita.
+
+Como entregable final tendrás una página web que permita **visualizar la data,
+filtrarla, ordenarla y hacer algún cálculo agregado**. Con cálculo agregado
+nos referimos a distintos cálculos que puedes hacer con la data para mostrar
+información aún más relevante para los usuarios (promedio, el valor máximo
+o mínimo, etc).
+
+En este proyecto nos iremos familiarizando con el uso de herramientas de IA
+(inteligencia artificial) como chatGPT o herramientas de [Stable Diffussion](https://es.wikipedia.org/wiki/Stable_Diffusion).
+Así que tu tarea en este proyecto
+será pedirle a la IA un set de datos en formato [JSON](https://www.json.org/json-es.html)
+para que lo puedas consumir en tu proyecto y mostrar la data en una página web.
+
+La temática será a tu gusto, por ejemplo, pueden ser personajes importantes en la
+historia, países, películas... etc. La estructura de este archivo JSON deberá ser
+la siguiente:
+
+* `name:` Nombre del dato a dibujar en el front.
+* `short-description:` Descripción corta sobre el dato.
+* `img:` URL de la imagen. Esta imagen será generada a través del método Stable
+  diffussion. Una vez generada la imagen, ésta será descargada y guardada en
+  una carpeta del proyecto. La URL de esta imagen será agregada manualmente al
+  JSON generado por la AI. Para generar esta imagen, te recomendamos esta
+  [herramienta](https://huggingface.co/spaces/nota-ai/compressed-stable-diffusion).
+* `number:` Un valor numérico para el cálculo agregado (por ejemplo,
+  puntuaciones, cantidades, etc).
+* `number-description:` Descripción de lo que representa el número anterior.
```

Comment:
`statistic` and `statistic-description` ? `statistic-name` o `statistic-label` seria mejor?

o que te parece `statistic`: { label: 'el nombre', value: 2.0 }